af/shovel/centrifugation.py

```diff
@@ -35,6 +36,8 @@
 # causing bucket-wide exception, so re-import is not forced.
 CODE_VER = 4
 
+CODE_VER_REPROCESS = 0 # special `code_ver` value to do partial re-processing
```

Comment:
Can you expand this comment explaining the use case of `CODE_VER_REPROCESS`? Documentation on it's usage should also be added to the README docs.

Reply:
It's actually there, but deserves some more words:

```diff
+Both these cases are handled with `UPDATE autoclaved SET code_ver = 0 WHERE …`.
+`centrifugation.py` will re-ingest alike autoclaved file while preserving `msm_no`.
```